docs/zh/api-reference/rerank-api.md

```
@@ -0,0 +1,227 @@
+    "深度学习使用神经网络...",
+    "统计学涉及数据收集和分析..."
+  ],
+  "top_n": 2,
+  "return_documents": true
+}
+```
+
+**参数：**
+
+| 参数 | 类型 | 必需 | 描述 |
+|------|------|------|------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)